DorisProject/src/environments/environment.ts

```diff
@@ -11,7 +11,8 @@ export const environment = {
   updateProduct: '/api/updateProduct',
   getProductSku: '/api/getProduct/',
   getCategoryList: '/api/getCategoryList/',
-  saveCategory: '/api/category'
+  saveCategory: '/api/category',
+  deleteCategory: '/api/deleteCategoryByIdJpa'
```

Comment:
Codacy has a fix for the issue: [Insert `,`](https://app.codacy.com/gh/SaraArenasV/DorisProjectFrontEnd/pullRequest?prid=9277326)
```suggestion
  deleteCategory: '/api/deleteCategoryByIdJpa',
```